els/frontend/src/utils/helpers/authHelper.js

```diff
@@ -1,13 +1,10 @@
+import CONFIG from '../const';
 // Setup config with token - helper function
```

Comment:
Put a space after doing importing
```suggestion
import CONFIG from '../const';

// Setup config with token - helper function
```